crates/tabby/src/main.rs

```diff
@@ -40,6 +42,11 @@ pub enum Commands {
     /// Run scheduler progress for cron jobs integrating external code repositories.
     Scheduler(SchedulerArgs),
 
+    /// Run specified job in background
+    #[cfg(feature = "ee")]
+    #[clap(name = "job::sync", hide = true)]
+    JobSync(job::SyncArgs),
```

Comment:
job::sync and job::index each should be individual command, just like WorkerCompletion/WorkerChat